add gnICE USB ids to the FTDI drivers

--- ftdibus.inf
+++ ftdibus.inf
@@ -49,6 +49,8 @@
 %USB\VID_0403&PID_6011&MI_01.DeviceDesc%=FtdiBus.NT,USB\VID_0403&PID_6011&MI_01
 %USB\VID_0403&PID_6011&MI_02.DeviceDesc%=FtdiBus.NT,USB\VID_0403&PID_6011&MI_02
 %USB\VID_0403&PID_6011&MI_03.DeviceDesc%=FtdiBus.NT,USB\VID_0403&PID_6011&MI_03
+%gnICE_Hw%=FtdiBus.NT,USB\VID_0456&PID_f000&MI_01
+%gnICE+_Hw%=FtdiBus.NT,USB\VID_0456&PID_f001&MI_01
 
 [FtdiHw.NTamd64]
 %USB\VID_0403&PID_6001.DeviceDesc%=FtdiBus.NTamd64,USB\VID_0403&PID_6001
@@ -58,6 +60,8 @@
 %USB\VID_0403&PID_6011&MI_01.DeviceDesc%=FtdiBus.NTamd64,USB\VID_0403&PID_6011&MI_01
 %USB\VID_0403&PID_6011&MI_02.DeviceDesc%=FtdiBus.NTamd64,USB\VID_0403&PID_6011&MI_02
 %USB\VID_0403&PID_6011&MI_03.DeviceDesc%=FtdiBus.NTamd64,USB\VID_0403&PID_6011&MI_03
+%gnICE_Hw%=FtdiBus.NT,USB\VID_0456&PID_f000&MI_01
+%gnICE+_Hw%=FtdiBus.NT,USB\VID_0456&PID_f001&MI_01
 
 [ControlFlags]
 ExcludeFromSelect=*
@@ -127,3 +131,5 @@
 USB\VID_0403&PID_6011&MI_03.DeviceDesc="USB Serial Converter D"
 SvcDesc="USB Serial Converter Driver"
 ClassName="USB"
+gnICE_Hw="gnICE USB Serial Converter"
+gnICE+_Hw="gnICE+ USB Serial Converter"
--- ftdiport.inf
+++ ftdiport.inf
@@ -42,11 +42,15 @@
 %VID_0403&PID_6001.DeviceDesc%=FtdiPort232.NT,FTDIBUS\COMPORT&VID_0403&PID_6001
 %VID_0403&PID_6010.DeviceDesc%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0403&PID_6010
 %VID_0403&PID_6011.DeviceDesc%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0403&PID_6011
+%gnICE_UART%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0456&PID_F000
+%gnICE+_UART%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0456&PID_F001
 
 [FtdiHw.NTamd64]
 %VID_0403&PID_6001.DeviceDesc%=FtdiPort232.NTamd64,FTDIBUS\COMPORT&VID_0403&PID_6001
 %VID_0403&PID_6010.DeviceDesc%=FtdiPort2232.NTamd64,FTDIBUS\COMPORT&VID_0403&PID_6010
 %VID_0403&PID_6011.DeviceDesc%=FtdiPort2232.NTamd64,FTDIBUS\COMPORT&VID_0403&PID_6011
+%gnICE_UART%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0456&PID_F000
+%gnICE+_UART%=FtdiPort2232.NT,FTDIBUS\COMPORT&VID_0456&PID_F001
 
 [FtdiPort.NT.AddService]
 DisplayName    = %SvcDesc%
@@ -160,5 +164,7 @@
 VID_0403&PID_6011.DeviceDesc="USB Serial Port"
 SvcDesc="USB Serial Port Driver"
 SerEnum.SvcDesc="Serenum Filter Driver"
+gnICE_UART="gnICE (UART)"
+gnICE+_UART="gnICE+ (UART)"
 
 
