Index: buildscript-experimental/toolchain-regtest
===================================================================
--- buildscript-experimental/toolchain-regtest	(revision 3211)
+++ buildscript-experimental/toolchain-regtest	(working copy)
@@ -1945,27 +1945,6 @@
 	test_ld $1
     fi
 
-    if [ $TEST_GDB -eq 1 ] ; then
-	# Get rid of old Xvfb & rsh processes, which might be lurking around from old runs
-	RUN=`ps aux | grep Xvfb | grep -v grep | wc -l`
-	if [ $RUN -gt 0 ] ; then
-            echo "Found old Xvfb process, killing it"
-            RUN="ps aux | grep Xvfb | grep -v grep ; killall -9 Xvfb"
-            (echo "echo -e '###\n$RUN\n' $LOGFILE" ) | sh
-            (echo "($RUN) $LOGFILE" ) | sh
-	fi
-	RUN=`ps aux | grep rsh | grep -v grep | wc -l`
-	if [ $RUN -gt 0 ] ; then
-            echo "Found old rsh process, killing it"
-            RUN="ps aux | grep rsh | grep -v grep ; killall -9 rsh"
-            (echo "echo -e '###\n$RUN\n' $LOGFILE" ) | sh
-            (echo "($RUN) $LOGFILE" ) | sh
-	fi
-
-	clean_hardware $2 gdb
-	test_gdb $1
-    fi
-
     if [ $TEST_GCC -eq 1 ] ; then
 	clean_hardware $2 gcc
 	test_gcc gcc $1 "$GCC_TESTSUITE"
@@ -2007,6 +1986,27 @@
 	test_libgomp $1 $2
     fi
 
+    if [ $TEST_GDB -eq 1 ] ; then
+	# Get rid of old Xvfb & rsh processes, which might be lurking around from old runs
+	RUN=`ps aux | grep Xvfb | grep -v grep | wc -l`
+	if [ $RUN -gt 0 ] ; then
+            echo "Found old Xvfb process, killing it"
+            RUN="ps aux | grep Xvfb | grep -v grep ; killall -9 Xvfb"
+            (echo "echo -e '###\n$RUN\n' $LOGFILE" ) | sh
+            (echo "($RUN) $LOGFILE" ) | sh
+	fi
+	RUN=`ps aux | grep rsh | grep -v grep | wc -l`
+	if [ $RUN -gt 0 ] ; then
+            echo "Found old rsh process, killing it"
+            RUN="ps aux | grep rsh | grep -v grep ; killall -9 rsh"
+            (echo "echo -e '###\n$RUN\n' $LOGFILE" ) | sh
+            (echo "($RUN) $LOGFILE" ) | sh
+	fi
+
+	clean_hardware $2 gdb
+	test_gdb $1
+    fi
+
     clean_hardware $2 done
 
     if [ $LEGACY_RESULT ] ; then
